test-stats.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+# Copyright Red Hat Inc
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# This file is part of Cockpit.
+
+import argparse
+from tools import *
+
+from generate_file_upload_wiki import generate_file_upload_on_wiki
+
+
+UPSTREAMFIRST_URL = config.UPSTREAMFIRST_URL
```

Comment:
Why do you do this?? Do you want to change UPSTREAMFIRST_URL? If no, why assign it again?

Reply:
Deleted it. This string is not used.